test/utils.ts

```diff
@@ -72,3 +72,31 @@ export function humanFileSize(bytes, si = false, dp = 1) {
 
   return bytes.toFixed(dp) + ' ' + units[u];
 }
+
+/**
+ * Check if the current platform is Windows
+ *
+ * @returns boolean
+ */
+export function isWindowsOperatingSystem(): boolean {
+  return platform().indexOf('win') === 0;
+}
+
+/**
+ * Conditionally run a test if the condition is true
+ *
+ * @param condition boolean
+ * @returns jest.It
+ */
+export function testIf(condition: boolean): jest.It {
+  return condition ? test : test.skip;
+}
+
+/**
+ * Conditionally run a describe block if the condition is true
+ *
+ * @param condition boolean
+ * @returns jest.Describe
```

Comment:
IMHO for a better understanding what this method does.

```suggestion
 * @returns A Jest `Describe` function reference you can call like `describeIf(cond)('suite', fn)`.
 * 
 * @example
 * describeIf(!isWindowsOperatingSystem())('non-Windows suite', () => {
 *   it('works', () => { /* ... */ });
 * });
```

If you could add the same for all these three want you added

Reply:
Thank you for bringing up the idea of ​​adding a @returnstag to the docblock for consistency. While I'm politely declining the addition of that specific @returnstag for redundancy reasons (since the return type is clear in the TypeScript signature and the @example), I really appreciate you highlighting the broader need for consistency in our documentation.

Your suggestion underscores the value of adopting a docblock linting standard in the future. Having a linter would allow us to enforce a uniform set of rules, ensuring every docblock—whether for describeIf, itIf, or others—meets a predictable quality standard. This would help prevent the very inconsistency we're discussing and make the entire API documentation much clearer. I'll make a note that standardising our docblocks with a linter is something we should consider for a future follow up.